### ISSUE TITLE
parallelization not needed for decoder tool

### ISSUE THREAD

**Issue by teddythepooh** [teddythepooh/NIBRS-Master-Files-Decoder#6]
As a Research Analyst at the University of Chicago Crime Lab (at the time that I am writing this issue), I routinely leverage parallel processing through Python's `multiprocessing` module (e.g., simulation-based power analyses on multiple outcomes); and in most cases through our HPC environment (SLURM). However, I have also found success with executing my Python or R scripts in bash using the `&`  command. For example, I had a script that intakes and processes arrests data for six or so police agencies; the runtimes are three and six minutes with and without `&`.

I thought the same case would apply to this project. I had set up the bash script to run my Python decoder on four NIBRS segments in parallel. After 30 minutes, my program crashed due to a memory error. After I removed the `&`  command, I deployed the decoder tool again and it only took nine minutes.

![image](https://github.com/user-attachments/assets/c69eb943-17bf-45ed-b856-3af0889a0ded)
